multiqc/modules/bases2fastq/plot_runs.py

```diff
@@ -0,0 +1,326 @@
+        )
+        num_polonies[run] = num_polonies_run
+
+        total_yield_run = OrderedDict()
+        total_yield = run_data[run]["TotalYield"]
+        total_yield_run["Perfect Index"] = total_yield * percent_perfect_total * 0.01
+        total_yield_run["Mismatched Index"] = total_yield * percent_imperfect_total * 0.01
+        total_yield_run["Unassigned"] = (
+            total_yield - total_yield_run["Perfect Index"] - total_yield_run["Mismatched Index"]
+        )
+        yields[run] = total_yield_run
+
+    plot_content = [num_polonies, yields]
+    config = {
+        "data_labels": [
+            {"name": "Number of Polonies", "ylab": "Number of Polonies", "format": "{d}"},
+            {"name": "Yield (Gb)", "ylab": "Gb"},
+        ],
+        "cpswitch": True,
+        "stacking": "normal",
+        "description": "Bar plots to compare yields of all found sequencing runs.",
+        "id": "run_metrics_bar",
+        "title": "bases2fastq: General Sequencing Run QC metrics plot",
+        "ylab": "QC",
+    }
+    cats = [
+        {
+            "Perfect Index": {"name": "Perfect Index", "color": "#7cb4ec"},
+            "Mismatched Index": {"name": "Mismatched Index", "color": "#ff7518"},
+            "Unassigned": {"name": "Unassigned Index", "color": "#434348"},
+        }
+    ] * 2
+
+    plot_name = "Sequencing Run Yield"
+    plot_html = bargraph.plot(plot_content, cats, pconfig=config)
+    anchor = "run_yield_plot"
+    description = "Bar plots of sequencing run yields. Please see individual run reports for details"
+    helptext = """
+    This section shows and compare the yield and index assignment rate of each sequencing run.\n\n
+       - Number of Polonies: The (total) number of polonies calculated for the run\n
+       - Yield: The run yield based on assigned reads in gigabases
+    """
+    return plot_html, plot_name, anchor, description, helptext, plot_content
+
+
+def tabulate_run_stats(run_data, color_dict):
+    """
+    Tabulate general information and statistics of each run
+    """
+    plot_content = dict()
+    for s_name in run_data.keys():
+        run_stats = dict()
+        run_stats.update({"num_polonies_run": run_data[s_name]["NumPolonies"]})
+        run_stats.update({"percent_assigned_run": run_data[s_name]["PercentAssignedReads"]})
+        run_stats.update({"yield_run": run_data[s_name]["AssignedYield"]})
+        run_stats.update({"mean_base_quality_run": run_data[s_name]["QualityScoreMean"]})
+        run_stats.update({"percent_q30_run": run_data[s_name]["PercentQ30"]})
+        run_stats.update({"percent_q40_run": run_data[s_name]["PercentQ40"]})
+        plot_content.update({s_name: run_stats})
+
+    headers = OrderedDict()
+
+    headers["num_polonies_run"] = {
+        "title": "Number of Polonies",
+        "format": "{d}",
+        "description": "The (total) number of polonies calculated for the run",
+        "min": 0,
+        "scale": "RdYlGn",
```

Comment:
Just repeating some code comments that might be lost now :)

`#Polonies` should be rounded to [read or bases counts](https://github.com/ewels/MultiQC/blob/41c1c85813c27a6cec2158d664594afacf178417/multiqc/utils/config_defaults.yaml#L29-L37C28) in the general stats table. See how [it's done](https://github.com/ewels/MultiQC/blob/master/multiqc/modules/bcl2fastq/bcl2fastq.py#L394-L401
) for the bcl2fastq module as a reference.

Reply:
Same for other metrics. Could you take a look if similar metrics that are already reported in other modules, and try to use the same naming style, value formatting, and color scheme. E.g. [bcl2fastq has](https://github.com/ewels/MultiQC/blob/master/multiqc/modules/bcl2fastq/bcl2fastq.py#L462C1-L469) Q30, yield, mean base quality. FastQC can be a reference for other metrics and plots.